### PR TITLE
updating image format in olm create bundle to match the new image format

### PR DESCRIPTION
**Pull request by acornett21**
Issue #, if available:
fixes aws-controllers-k8s/community#970

Description of changes:
Update `olm-create-bundle` script to follow new image formate. This code follows the same format/intent as the `containerImage` template code.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/olm-create-bundle.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT_DIR="$SCRIPTS_DIR/.."
 BUILD_DIR="$ROOT_DIR/build"
 DEFAULT_BUNDLE_CHANNEL="alpha"
-DEFAULT_SERVICE_CONTROLLER_CONTAINER_REPOSITORY="public.ecr.aws/aws-controllers-k8s/controller"
+DEFAULT_SERVICE_CONTROLLER_CONTAINER_REPOSITORY="public.ecr.aws/aws-controllers-k8s/"
```
Comment (vijtrip2) [inline scripts/olm-create-bundle.sh:12]: nit
```suggestion
DEFAULT_SERVICE_CONTROLLER_CONTAINER_REGISTRY="public.ecr.aws/aws-controllers-k8s"
```

File: scripts/olm-create-bundle.sh [hunk 2]
```diff
@@ -114,7 +114,7 @@ if [ -n "$AWS_SERVICE_DOCKER_IMG" ] && [ -n "$SERVICE_CONTROLLER_CONTAINER_REPOS
 fi
 
 SERVICE_CONTROLLER_CONTAINER_REPOSITORY=${SERVICE_CONTROLLER_CONTAINER_REPOSITORY:-$DEFAULT_SERVICE_CONTROLLER_CONTAINER_REPOSITORY}
-DEFAULT_AWS_SERVICE_DOCKER_IMAGE="$SERVICE_CONTROLLER_CONTAINER_REPOSITORY:$SERVICE-$VERSION"
+DEFAULT_AWS_SERVICE_DOCKER_IMAGE="$SERVICE_CONTROLLER_CONTAINER_REPOSITORY$SERVICE-controller:v$VERSION"
```
Comment (vijtrip2) [inline scripts/olm-create-bundle.sh:117]: nit
```suggestion
DEFAULT_AWS_SERVICE_DOCKER_IMAGE="$SERVICE_CONTROLLER_CONTAINER_REGISTRY/$SERVICE-controller:v$VERSION"
```

**Review by vijtrip2 — COMMENTED** [review]
/ok-to-test

File: scripts/olm-create-bundle.sh (hunk 2 above)
Comment (acornett21) [reply scripts/olm-create-bundle.sh]: fixed

**Pull request merged**